hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestSpaceQuotaDropTable.java

```diff
@@ -87,6 +94,50 @@ public void testSetQuotaAndThenDropTableWithDisable() throws Exception {
     setQuotaAndThenDropTable(SpaceViolationPolicy.DISABLE);
   }
 
+  @Test
+  public void testSetQuotaAndThenDropTableWithRegionReport() throws Exception {
```

Comment:
Great job at capturing this in a concise test case.

Reply:
Thanks for the review @joshelser . Did all the changes and pushed. :)